Eldev

```diff
@@ -78,7 +78,10 @@ be executed via `install.sh'."
         (eldev-print "  %s" directory)))
     (eldev-print "\nIf you use a different directory, make sure to add it to `PATH'\nenvironment variable.")
     (signal 'eldev-quit 1))
-  (let ((directory (car parameters)))
+  (let ((directory (car parameters))
+        (shell-script-name (if (equal system-type 'windows-nt)
```

Comment:
Probably could be broken out into a package-private function, I see you are using this several times.

Reply:
Introduced `eldev.util:eldev--is-windows-nt` fn.

(or did you perhaps mean moving the whole `if` form to a new function?)

Reply:
Yes, move the whole `(if ...` into a function of its own, e.g. `eldev--shell-script-name` or sth. like that.

Reply:
Done.